pkg/flag/misconf_flags.go

```diff
@@ -13,6 +13,12 @@ var (
 		Value:      false,
 		Usage:      "remove policy bundle",
 	}
+	ScanAllDirsFlag = Flag{
+		Name:       "scan-all-dirs",
+		ConfigName: "misconfiguration.scan-all-dirs",
+		Value:      true,
+		Usage:      "scan all dirs recursively",
+	}
```

Comment:
Another question: do we need the flag? We can just change the default behavior and add this flag when someone complains about it. I'm wondering if almost all users want to enable it. Please correct me if I'm wrong.

Reply:
> This would at least give users the time to fix all the issues before their pipeline is operational again

It also makes sense. We can probably explain that Trivy had a bug on Terraform scanning and didn't work well. They can keep using older versions until they fix issues.

Reply:
+1 for just making this the default -- it really feels like the right thing for the tool.